test/powershell/Modules/Microsoft.PowerShell.Management/Get-PSProvider.Tests.ps1

```diff
@@ -24,4 +24,64 @@ Describe "Get-PSProvider" -Tags "CI" {
 
 	{ $actual | Format-List } | Should -Not -Throw
     }
+
+    Context 'PathSeparators' {
+        BeforeAll {
+            $testCases = if ($IsWindows) {
+                            @(
+                                @{Provider = 'FileSystem'; Value = @("\", "/")}
+                                @{Provider = 'Variable'; Value = @("\", "/")}
+                                @{Provider = 'Function'; Value = @("\", "/")}
+                                @{Provider = 'Alias'; Value = @("\", "/")}
+                                @{Provider = 'Environment'; Value = @("\", "/")}
+                                @{Provider = 'Certificate'; Value = @("\", "/")}
+                                @{Provider = 'Registry'; Value = @("\")}
+                            )
+                        }
+                        else {
+                            @(
+                                @{Provider = 'FileSystem'; Value = @("/", "\")}
+                                @{Provider = 'Variable'; Value = @("/", "\")}
+                                @{Provider = 'Function'; Value = @("/", "\")}
+                                @{Provider = 'Alias'; Value = @("/", "\")}
+                                @{Provider = 'Environment'; Value = @("/", "\")}
+                                @{Provider = 'Certificate'; Value = @("/", "\")}
```

Comment:
The `Certificate` and `WSMan` providers are windows-only.

Reply:
I have removed them from the nonWindows case. I realized at that point that the `WSMan` provider is not enabled when these tests are run. Should I add the corresponding tests for `WSMan` at [ConfigProvider.Tests.ps1](https://github.com/PowerShell/PowerShell/blob/master/test/powershell/Modules/Microsoft.WSMan.Management/ConfigProvider.Tests.ps1)?

Reply:
If `WSMan` provider is changed in this PR, then it should also be tested along with the rest.

Reply:
Added tests for `ItemSeparator`/`AltItemSeparator` at https://github.com/PowerShell/PowerShell/pull/8587/files#diff-c2f140f155813763bf7a3492a6a03196